### PR TITLE
chore(TabList): Add Tab as a subcomponent of TabList in the storybook docsite

### PR DESCRIPTION
**Pull request by behowell**
## Previous Behavior

The `Tab` component isn't documented anywhere on the [FluentUI storybook](https://react.fluentui.dev/).

## New Behavior

Add `Tab` as a subcomponent of `TabList`.

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/Avatar-mount_base.html">632</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/Avatar-mount.html">647</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/Button-mount_base.html">295</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/Button-mount.html">303</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/Field-mount_base.html">1136</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/Field-mount.html">1110</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/FluentProvider-mount_base.html">709</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/FluentProvider-mount.html">732</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/FluentProviderWithTheme-mount_base.html">74</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/FluentProviderWithTheme-mount.html">84</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">59</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">64</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">72</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">77</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/MakeStyles-mount_base.html">861</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/MakeStyles-mount.html">866</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/Persona-mount_base.html">1752</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/Persona-mount.html">1707</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/SpinButton-mount_base.html">1362</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30794/perf-test-react-components/SpinButton-mount.html">1395</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports | Size (minified/GZIP) |
| ----------------- | -------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr> | `71.098 kB`<br />`20.515 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `217.205 kB`<br />`61.941 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr> | `43.585 kB`<br />`14.352 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr> | `7.944 kB`<br />`2.588 kB` |
</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/b2ded2504c2579038b15ca797666cc81d55c0364'>b2ded2504c2579038b15ca797666cc81d55c0364</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-components/react-tabs/stories/Tabs/index.stories.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { TabList } from '@fluentui/react-components';
+import { Tab, TabList } from '@fluentui/react-components';
```
Comment (fabricteam) [inline packages/react-components/react-tabs/stories/Tabs/index.stories.tsx:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30794&commitId=f3d05880a8a8afe2cc4bca55636c1fbea23d228e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Image Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image Converged.Image Layout Fit.normal.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Review by GeoffCoxMSFT — APPROVED** [review]

**Pull request merged**